en/docs/remote-apps/connect-remote-app.md

```diff
@@ -0,0 +1,123 @@
+
+6. Copy the Ballerina configurations.
+
+Now you have the application secret that you need to set up the Ballerina project of your remote application.
+
+## Step 2: Set up the Ballerina project
+
+In this step, you will set up your Ballerina project with the configurations that connect the Ballerina application to Choreo. Here, the Ballerina project is the directory that contains the definition of your Ballerina application in a `.bal` file and all the configuration files required to run it.
+
+To set up the Ballerina Project:
+
+1. Create a directory in your machine (e.g., named `HelloWorld`) and move your file with the remote ballerina application (in this example the `hello_service.bal` file) to it.
+
+    !!! info
+        This directory will be referred to as `<BALLERINA_PROJECT>`from here onward.
+
+2. To link the Ballerina application to Choreo via the application secret, open a new file in a text editor and add the Ballerina configurations you copied in Step 1. Then save it as `Config.toml` in the `<BALLERINA_PROJECT>`.
+
+3. To enable the Ballerina application to publish observability statistics, save a file named `Ballerina.toml` with the following content in the `<BALLERINA_PROJECT>`.
+
+    ```ballerina
+    [build-options]
+    observabilityIncluded = true
+    ```
+   
+4. For the Ballerina application to import the `ballerinax/choreo` module, open the `.bal` file (in this example, `hello_service.bal`) and at the following line at the top.
+
+    ```ballerina
+    import ballerinax/choreo as _;
+    ```
+   
+Now you have completed setting up your Ballerina project. It will connect to Choreo once you build and run it.
+
+## Step 3: Build the Ballerina project
+
+In this step, you will build and run your Ballerina project. 
+
+1. To build the Ballerina project, open your terminal, navigate to `<BALLERINA_PROJECT>`, and issue the following command.
+
+    ```text
+    bal build
+    ```
+   
+    This creates a `JAR` in the `<BALLERINA_PROJECT>/target/bin` sub directory.
+   
+2. To run the Ballerina application, issue the following command from the `<BALLERINA_PROJECT>`.
+
+    ```
+    bal run target/bin/<JAR_NAME>.jar
+    ```
+   
+    e.g., `bal run target/bin/HelloService.jar`
+    
+    Logs similar to the following appear in the terminal.
+    
+    ```
+    ballerina: initializing connection with observability backend periscope.choreo.dev:443
+    ballerina: visit http://console.choreo.dev/observe/app/91e9750f-e2e1-11eb-90ce-de6d84510939/dd9780a8-e2e2-11eb-90ce-de6d84510939 to access observability data
+    ballerina: started publishing metrics to Choreo
+    [ballerina/http] started HTTP/WS listener 0.0.0.0:9090
+    ballerina: started publishing traces to Choreo
+    ballerina: sending all remaining traces to Choreo
+    [ballerina/http] stopped HTTP/WS listener 0.0.0.0:9090
+    ballerina: sending metrics to Choreo
```

Comment:
we may need to remove these three lines, as they are executed once we stop the Ballerina service.
In Step 4, we mention about invoking the service, therefore its better to have the output at the running state.

Reply:
Done